demo/nuxt.config.ts

```diff
@@ -1,27 +1,25 @@
 export default defineNuxtConfig({
   modules: [
     // https://github.com/nuxt-modules/supabase
-    '@nuxtjs/supabase',
-    // UI lib (will be soon open sourced)
-    '@nuxthq/ui',
+    "@nuxtjs/supabase",
+     // https://github.com/nuxt/ui
+     "@nuxt/ui",
     // https://github.com/nuxt-modules/color-mode
-    '@nuxtjs/color-mode',
+    "@nuxtjs/color-mode",
   ],
+
   runtimeConfig: {
     public: {
-      baseUrl: process.env.BASE_URL || 'http://localhost:3000',
+      baseUrl: process.env.BASE_URL || "http://localhost:3000",
     },
   },
-  ui: {
-    colors: {
-      primary: 'green',
-    },
-    icons: ['mdi', 'heroicons', 'heroicons-outline'],
-  },
+
   supabase: {
     redirectOptions: {
-      login: '/',
-      callback: '/confirm',
+      login: "/",
+      callback: "/confirm",
     },
   },
-})
+
+  compatibilityDate: "2024-09-01",
+});
```

Comment:
```suggestion
export default defineNuxtConfig({
  modules: [
    // https://github.com/nuxt-modules/supabase
    '@nuxtjs/supabase',
    // https://github.com/nuxt/ui
    '@nuxt/ui',
    // https://github.com/nuxt-modules/color-mode
    '@nuxtjs/color-mode',
  ],

  runtimeConfig: {
    public: {
      baseUrl: process.env.BASE_URL || 'http://localhost:3000',
    },
  },

  supabase: {
    redirectOptions: {
      login: '/',
      callback: '/confirm',
    },
  },

  compatibilityDate: '2024-09-01',
})
```

Reply:
Thanks for the review! Just now realized there's a linter in the root of the repo 🙈 fixed